src/modules/facility-fixed-fee/facility-fixed-fee.controller.ts

```diff
@@ -23,7 +34,55 @@ export class FacilityFixedFeeController {
   @ApiInternalServerErrorResponse({
     description: 'An internal server error has occurred.',
   })
-  async getFixedFeesForFacility(@Param() params: GetFacilityFixedFeeParamsDto): Promise<GetFacilityFixedFeeResponse> {
+  async getFixedFeesForFacility(@Param() params: FacilityFixedFeeParamsDto): Promise<GetFacilityFixedFeeResponse> {
     return await this.facilityFixedFeeService.getFixedFeesForFacility(params.facilityIdentifier);
   }
+
+  @Post('facilities/:facilityIdentifier/fixed-fees')
+  @ApiOperation({
+    summary: 'Create a new fixed fee for a facility.',
+  })
+  @ApiBody({
+    type: CreateFacilityFixedFeeRequestItem,
+    isArray: true,
+  })
+  @ApiCreatedResponse({
+    description: 'The fixed fee has been successfully created.',
+    type: CreateFacilityFixedFeeResponse,
+  })
+  @ApiNotFoundResponse({
+    description: 'The facility was not found.',
+  })
+  @ApiBadRequestResponse({
+    description: 'Bad request.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  async createFixedFeeForFacility(
+    @Param() params: FacilityFixedFeeParamsDto,
+    @Body(new ParseArrayPipe({ items: CreateFacilityFixedFeeRequestItem }))
```

Comment:
We have a new `ValidatedArrayBody` decorate we can use for this input

Reply:
Changed to use ValidatedArrayBody